src/components/Books/BookProgress.jsx

```diff
@@ -1,16 +1,20 @@
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const BookProgress = () => {
+  const percentage = 64;
+
   return (
     <div className="scdDiv">
       <div className="BookStatus">
-        <div className="progressBar">
-          <div>64%</div>
+        <div>
+          <CircularProgressbar value={percentage} text={''} />
         </div>
-        <span className="headSpan" />
         <div>
-          <h3>
-            64% <br />
-            <span>Completed</span>
+          <h3 className="percentage">
+            {percentage}%
+            <br />
+            <span className="completed">Completed</span>
```

Comment:
- [ ]  As the previous code reviewer mentioned related to your design I am also insisting you make your design **EXACTLY** look like the *Zeplin* design. Kindly do so.
   - This should include the colors, font size, font family, spacing e.t.c.

Your design
![Screenshot from 2023-09-03 00-36-07](https://github.com/GeekyHacks/BookStoreReact_Vite/assets/66476329/c2a040e6-3900-4904-8fc7-59c494808c04)


Given design 
![Screenshot from 2023-09-03 00-37-49](https://github.com/GeekyHacks/BookStoreReact_Vite/assets/66476329/697f73d0-3f7c-402f-a2ef-16484a37963a)